src/xenia/kernel/xam_module.h

```diff
@@ -24,7 +24,25 @@ class XamModule : public XKernelModule {
 
   static void RegisterExportTable(xe::cpu::ExportResolver* export_resolver);
 
+  struct LoaderData {
+    LoaderData() {
+      launch_data_ptr = 0;
+      launch_data_size = 0;
+      launch_flags = 0;
+      launch_path = "";
+    }
+
+    uint32_t launch_data_ptr;
```

Comment:
(nit)
We can use C++ default initializers - I've fallen in love with them - so this would become:
```
struct LoaderData {
  uint32_t launch_data_ptr = 0;
  ...
  std::string launch_path; // no need to init, as already ""
};
```

I'm moving things over to this style as I see them.